content/appendices/templates.md

```diff
@@ -47,20 +47,30 @@ The package includes all the following forms of documentation:
 Readme requirements
 The package meets the readme requirements below:
 
-- [ ] Package has a README.md file in the root directory. 
+- [ ] Package has a user-friendly README.md file in the root directory. 
 
 The README should include, from top to bottom:
```

Comment:
how strict do we want to be on the "top to bottom"? For example, if the readme has a badge for the docs, do we also require a heading that points to the docs? Personally, I would be inclined to leave this simply at "The README should include" and give the authors some discretion to decide the exact order.

Reply:
I'm not OK with:

- Badges for test coverage and repostatus.org badge.

We can recommend those but requiring them does not really add quality to the scientific package. While adding them does add a maintenance burden to the package author.


- descriptive links to all vignettes (rendered, i.e. readable, cf the documentation website section)

That creates some redundancy with the docs and I'm not a fan. It get stale very quickly and increases the maintenance burden.


- If applicable, how the package compares to other similar packages and/or how it relates to other packages

Nice to have but should not be mandatory.

- Citation information

Nice to have but should not be mandatory.

Reply:
i hear you both @lheagy @ocefpaf !! 
Ok so it sounds like
1. We should discuss badges! id definitely want a pyopensci one there which won't be any effort to maintain. but maybe we talk as a group about what other things we want to be visible. From my perspective, visibly showing some commitment to code coverage via testing and CI is important in a package. i could see that being presented in different ways. Thoughts on this?
2. A clean link to the docs via the readme. with an option of links to vignettes? i think @ocefpaf we are trying to figure out a way to say " it should be easy to find documentation and various parts of it specifically  (like vignettes) that a user would need to figure out how to use a package".  we are trying to leave space for a reviewer to provide usability input. if docs are hard to navigate we need to help submitters improve that usability component.
3. agreed... we ask for that relationship to other packages when the package is submitted. i agree it doesn't need to be mandatory. let's plan to chat more about this in our next meeting!! 
4. i think the citation information is very important. Especially with all of the issues now with scientific software not being "counted" towards scientists cv...in academic contexts. i would like to require that IF others agree.

Reply:
> definitely want a pyopensci one there which won't be any effort to maintain. 

:+1:

> From my perspective, visibly showing some commitment to code coverage via testing and CI is important in a package. i could see that being presented in different ways. Thoughts on this?

I see value in the coverage metric only for big projects with many contributors. Small projects with less people this value is lost b/c it is very easy to get 100% coverage and not tests what is important. That is why I prefer a policy of "recommend but do not require" when it comes to automated coverage. With that said, I do believe we should review packages paying attention to the coverage! We just should not outsource that to an automated tests/badge.

> A clean link to the docs via the readme. with an option of links to vignettes? i think @ocefpaf we are trying to figure out a way to say " it should be easy to find documentation and various parts of it specifically (like vignettes) that a user would need to figure out how to use a package". we are trying to leave space for a reviewer to provide usability input. if docs are hard to navigate we need to help submitters improve that usability component.

:+1: to adding links. I guess I'm not sure what 'descriptive links' mean? Maybe "clear links to the documentation" is clearer? (No pun indented.)

> i think the citation information is very important. Especially with all of the issues now with scientific software not being "counted" towards scientists cv...in academic contexts. i would like to require that IF others agree.

While I agree that it is very important it is a step that some research software engineers will not comply due to problems that may arise with the scientists in the team, institution policies, etc. We may close the door to important scientific software b/c some people may be stuck in old academia practices that will prevent them to generate a "citation" for the package. I can elaborate more (this is based on personal experience) at SciPy next week.